addons/bus/static/src/js/services/bus_service.js

```diff
@@ -58,29 +58,33 @@ var BusService =  CrossTab.extend(ServicesMixin, {
     /**
      * Send a notification, and notify once per browser's tab
      *
-     * @param {string} title
-     * @param {string} content
+     * @param {object} options
+     * @param {string} options.title
+     * @param {string} options.message
+     * @param {string} [options.type='warning'] 'info', 'success', 'warning', 'danger' or ''
```

Comment:
Type is not "warning" by default in this method, it is kept undefined. The default value is in `displayNotification`.